src/app/store/effects/VehicleTestResultModel.effects.spec.ts

```diff
@@ -29,12 +28,12 @@ describe('VehicleTechRecordModelEffects', () => {
       providers: [
         VehicleTestResultModelEffects,
         provideMockActions(() => actions),
-        {provide: TestResultService, useValue: TechnicalRecordServiceMock},
+        { provide: TestResultService, useValue: TechnicalRecordServiceMock },
         {
           provide: Store,
           useValue: {
             dispatch: jest.fn(),
-            pipe: jest.fn(() => hot('-a', {a: INITIAL_STATE})),
+            pipe: jest.fn(() => hot('-a', { a: INITIAL_STATE })),
             select: jest.fn()
           }
         }
```

Comment:
I think we will need to add unit tests covering the logic of effects depending on payload.vType we receive